domain-cc/cc-app/src/python_src/util/lookup_table.py

```diff
@@ -79,3 +144,54 @@ def get_lookup_table(filepath, input_key, output_key):
                 raise
 
     return classification_code_mappings
+
+
+def get_lookup_table(
+    version_num: str,
+    dropdown_v2_filepath: os.path,
+    dropdown_v1_filepath: os.path,
```

Comment:
is `os.path` the intended type?

![Screenshot 2023-11-30 at 11 26 05 AM](https://github.com/department-of-veterans-affairs/abd-vro/assets/17188013/55d2b642-7525-4f8b-969c-7f8159fc5bfd)